### DISCUSSION TITLE
Option to desactivate emoji suggestion

### DISCUSSION THREAD

**Discussion by pmattei** [community/community#11000] [Discussions]
Could you please provide an option to desactivate emoji suggestion?
Like another user said 

> Whenever one types colon “:” in github textboxes, an emoji pops up and worse: gets selected when pressing enter for newlines. Very annoying.

100% agree

I do not find any documentation on this subject

**Comment by jerstlouis** [comment]
This is the single most annoying thing on GitHub. Please go as far as disabling this option by default and/or improving it so that a user needs to actively *select* the emoji rather than automatically doing so while typing code. `:` are everywhere when typing code.

**Comment by G-Guillard** [comment]
See also [this question](https://stackoverflow.com/questions/50657698/how-can-i-stop-github-from-rendering-emojis-in-markdown) on StackOverflow.  The accepted answer is obviously non-satisfying.

To quote someone on [a similar issue](https://github.com/go-gitea/gitea/issues/11343) for the gitea project :

> 99.99999% of the time when I type a colon (:), I want to insert a colon. I don't ever want emoji in my issue descriptions, and the pop-up is super-annoying.

And to quote someone else on [another similar issue](https://github.com/overtake/TelegramSwift/issues/293) for the TelegramSwift project (yes, this must be an universaly-hated "feature"…) :

> An alternative would be to atleast not select any emoji by default when you type a colon. That is very annoying -- let it show me emoji but let me press enter and send my colon without automatically put 😍emoji in my work chat.

**Comment by fenugrec** [comment]
I'd be curious to parse all the comments, issues / PR text etc. for either of these patterns : 
- colon, newline, then list (like exactly this one )
- colon, then a quoted passage like the previous comment shows
- colon, then an inline link or other text

Probably a staggering number. In each of these instances, the writer was definitely annoyed by that ill-conceived emoji popup, and had to escape out of it, or delete it after an accidental insertion.
